### PR TITLE
Add hunting-buddy and t2 to setupaliases.lic

### PR DESCRIPTION
**Pull request by vtcifer**
as listed.

### CODE REVIEW

**Comment by vtcifer** [comment]
I don't see the trailing whitespace rubocop is mentioning here... if I read that right it should be on line 19?  Is it complaining about the extra CR/LF that was technically there before?

File: setupaliases.lic [hunk 1]
```diff
@@ -16,6 +16,15 @@
 
   # toggles training-manager on or off
   ['tm', "#{$clean_lich_char}eq Script.running?('training-manager') ? kill_script('training-manager') : start_script('training-manager')"],
+
```
Comment (MahtraDR) [inline setupaliases.lic:19]:
```suggestion

```

Comment (MahtraDR) [inline setupaliases.lic:19]:
```suggestion
 
```

Comment (MahtraDR) [reply setupaliases.lic:19]: Line 19 is a space. Not just a newline/carriage return.

File: setupaliases.lic [hunk 2]
```diff
@@ -16,6 +16,15 @@
...
```
Comment (MahtraDR) [inline setupaliases.lic:19]:
```suggestion

```

File: setupaliases.lic (hunk 1 above)
Comment (vtcifer) [reply setupaliases.lic]: Ah, when I went back to my repo in the web editor, I must have been on the wrong branch.  Thanks!

Comment (MahtraDR) [reply setupaliases.lic]: All good. :)

**Pull request merged**